### PR TITLE
support tman and other test frameworks.

### PR DESCRIPTION
**Pull request by zensh**

### CODE REVIEW

File: lib/run/index.js [hunk 1]
```diff
@@ -44,38 +38,53 @@ Runner.prototype.startServer = function() {
...
+  }).catch((err) => {
+    process.nextTick(() => {
+      throw err;
+    });
   });
 };
 
-Runner.prototype.initMocha = function() {
+Runner.prototype.initFramework = function() {
+  var frameworkBin = path.join(__dirname, '..', '..', 'node_modules', '.bin', this.options.framework);
```
Comment (zensh) [inline lib/run/index.js]: 这里还需要修一下，非默认 framework 与 macaca-cli 并不在同一个 node_modules 目录

Comment (zensh) [reply lib/run/index.js]: 修复，优先从项目目录（macaca 运行目录）加载测试框架

**Comment by xudafeng** [comment]
branch can not be merge auto, please rebase to resolve it, so sorroy. @zensh

**Comment by ziczhu** [comment]
👍 +1

**Pull request merged**